EmployeeMicroService/Data/Migrations/20240805193946_Added new field to cpature.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```

Comment:
something wrong with this

Reply:
I can see the code it was first migration